extract/extract/extractors/amazon_de.py

```diff
@@ -221,6 +221,27 @@ def parse_color(color: BeautifulSoup) -> Optional[str]:
     return _handle_parse(targets, parse_color)
 
 
+def _strip_image_url(url: str) -> str:
+    """
+    The images on product pages are usually resized and they might have overlays
+    like watermarks and play buttons for video thumbnails.
+    This function finds the url of the source image without additional processing applied.
+
+    Args:
+        url (str): the url of the image
+
+    Returns:
+        str: The url of the source image
+    """
+    scheme, netloc, path, query, fragment = urlsplit(url)
+    path_segments = path.split("/")
+    filename = path_segments[-1].split(".")
+    base_name = filename[0]
+    extension = filename[-1]
+    path_segments[-1] = f"{base_name}.{extension}"
```

Comment:
Just as a suggestion. there's a nice way to do this by using `pathlib.Path`
It exports the name and extension for you
```
from pathlib import Path
p = Path("/path/to/file/test.txt")
p.stem # outputs -> test
p.suffix # outputs -> .txt
```

Reply:
But just for my understanding the filename will be something in this format `some.file.name.and.more.name.txt"
And you'd return `some.txt`?

Maybe an example in the docs would be more helpful to understand what's happening with the image name